.gitignore

```diff
@@ -43,3 +43,4 @@ systemctl/ \*_/_.DS_Store
 typings
 *.log
 /abi/types/*
+web/dapp
```

Comment:
> 

I am not sure. but web/app is being generated in my side if `yarn start` --- you can check it out again and push the fixes

Reply:
> I understand this is probably our fault for not clearly documenting all this! Sorry we're working through things in real time thanks for your patience :)

I don't think its kinda your fault. I can honestly say I didn't go through the whole specs like codebase, scripts, naming schema, dependency schema, etc. I like your commenting. don't worry about my waiting and please write down the comments as much as possible.

Reply:
> > 
> 
> I am not sure. but web/app is being generated in my side if `yarn start` --- you can check it out again and push the fixes

yea we recently created some CI/CD scripts so this might be a vestige. It should only generate those directories when while Cloudflare/Fleek deploys so this is a bug we'll have to get around to fixing

Reply:
cool